app/javascript/packages/step-indicator/step-indicator-step.spec.tsx

```diff
@@ -43,7 +43,7 @@ describe('StepIndicatorStep', () => {
       );
 
       const title = getByText('Step');
-      const status = getByText('step_indicator.status.current');
+      const status = getByText('step_indicator.status.not_complete');
```

Comment:
Can we add a test case for the "current step" context?

Reply:
> Is there something you want covered in addition to lines 5-20 in this file?

Oh, good point! I totally missed we had test coverage for `StepStatus.CURRENT` already. I suppose it could be nice to have all of them covered (`PENDING` seems to be missing), but that's not in the scope of your work, so I wouldn't consider it a blocker.

Reply:
We were wondering about PENDING, and if it was really used. I'll go ahead and merge this, but there might be additional cleanup. Do you know what PENDING is for?

Reply:
It was originally used for marking the GPO letter (verify by mail) step as pending, but I seem to recall there may have been some recent revisions to the flow where we no longer show the pending state. Let me dig that up, since it would be nice if we could drop the extra logic (I had a hard time re-wrapping my head around the differences between "incomplete", "current", and "pending").

Reply:
It looks like the pending status was meant to have been removed as part of #6860. I can follow-up to clean up the lingering references, since I'm to blame for not having done it in the first place 😅